2022-01-26 06:38:36,178 INFO    MainThread:85210 [wandb_setup.py:_flush():71] setting env: {}
2022-01-26 06:38:36,178 INFO    MainThread:85210 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-26 06:38:36,178 INFO    MainThread:85210 [wandb_init.py:_log_setup():371] Logging user logs to /Users/kumagaihitoshi/Documents/GitHub/nd0821-c2-build-model-workflow-exercises/lesson-1-machine-learning-pipelines/demo/wandb/run-20220126_063836-3w23cr42/logs/debug.log
2022-01-26 06:38:36,178 INFO    MainThread:85210 [wandb_init.py:_log_setup():372] Logging internal logs to /Users/kumagaihitoshi/Documents/GitHub/nd0821-c2-build-model-workflow-exercises/lesson-1-machine-learning-pipelines/demo/wandb/run-20220126_063836-3w23cr42/logs/debug-internal.log
2022-01-26 06:38:36,178 INFO    MainThread:85210 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fd0c096f100>
2022-01-26 06:38:36,179 INFO    MainThread:85210 [wandb_init.py:init():404] calling init triggers
2022-01-26 06:38:36,179 INFO    MainThread:85210 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-26 06:38:36,179 INFO    MainThread:85210 [wandb_init.py:init():460] starting backend
2022-01-26 06:38:36,179 INFO    MainThread:85210 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-01-26 06:38:36,180 INFO    MainThread:85210 [backend.py:ensure_launched():216] starting backend process...
2022-01-26 06:38:36,188 INFO    MainThread:85210 [backend.py:ensure_launched():221] started backend process with pid: 85461
2022-01-26 06:38:36,189 INFO    MainThread:85210 [wandb_init.py:init():469] backend started and connected
2022-01-26 06:38:36,201 INFO    MainThread:85210 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-01-26 06:38:36,201 INFO    MainThread:85210 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-01-26 06:38:36,201 INFO    MainThread:85210 [wandb_init.py:init():533] updated telemetry
2022-01-26 06:38:36,224 INFO    MainThread:85210 [wandb_init.py:init():563] communicating current version
2022-01-26 06:38:37,312 INFO    MainThread:85210 [wandb_init.py:init():568] got version response 
2022-01-26 06:38:37,312 INFO    MainThread:85210 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-26 06:38:37,696 INFO    MainThread:85210 [wandb_init.py:init():606] starting run threads in backend
2022-01-26 06:38:40,247 INFO    MainThread:85210 [wandb_run.py:_console_start():1810] atexit reg
2022-01-26 06:38:40,247 INFO    MainThread:85210 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-01-26 06:38:40,247 INFO    MainThread:85210 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-01-26 06:38:40,248 INFO    MainThread:85210 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-26 06:38:40,248 INFO    MainThread:85210 [wandb_init.py:init():633] run started, returning control to user process
2022-01-26 06:38:40,249 INFO    MainThread:85210 [wandb_run.py:finish():1457] finishing run hitoshikumagai/multiple_runs/3w23cr42
2022-01-26 06:38:40,250 INFO    MainThread:85210 [jupyter.py:save_history():429] not saving jupyter history
2022-01-26 06:38:40,250 INFO    MainThread:85210 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-01-26 06:38:40,250 INFO    MainThread:85210 [wandb_init.py:_jupyter_teardown():303] cleaning up jupyter logic
2022-01-26 06:38:40,250 INFO    MainThread:85210 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-01-26 06:38:40,252 INFO    MainThread:85210 [wandb_run.py:_restore():1752] restore
2022-01-26 06:38:40,754 INFO    MainThread:85210 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  total_bytes: 1045
}

2022-01-26 06:38:41,141 INFO    MainThread:85210 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1045
  total_bytes: 1045
}

2022-01-26 06:38:41,739 INFO    MainThread:85210 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1045
  total_bytes: 15278
}

2022-01-26 06:38:41,844 INFO    MainThread:85210 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1045
  total_bytes: 15278
}

2022-01-26 06:38:41,952 INFO    MainThread:85210 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1045
  total_bytes: 15278
}

2022-01-26 06:38:42,058 INFO    MainThread:85210 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1045
  total_bytes: 15278
}

2022-01-26 06:38:42,163 INFO    MainThread:85210 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1045
  total_bytes: 15278
}

2022-01-26 06:38:42,270 INFO    MainThread:85210 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 15278
  total_bytes: 15278
}

2022-01-26 06:38:42,379 INFO    MainThread:85210 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 15278
  total_bytes: 15278
}

2022-01-26 06:38:42,486 INFO    MainThread:85210 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 15278
  total_bytes: 15278
}

2022-01-26 06:38:42,593 INFO    MainThread:85210 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 15278
  total_bytes: 15278
}

2022-01-26 06:38:42,701 INFO    MainThread:85210 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 15278
  total_bytes: 15278
}

2022-01-26 06:38:42,809 INFO    MainThread:85210 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 15278
  total_bytes: 15278
}

2022-01-26 06:38:43,238 INFO    MainThread:85210 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 15278
  total_bytes: 15278
}

2022-01-26 06:38:43,968 INFO    MainThread:85210 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 15278
  total_bytes: 15278
}
local_info {
}

2022-01-26 06:38:45,095 INFO    MainThread:85210 [wandb_run.py:_append_files():2180] logging synced files
